builtin/common/misc_helpers.lua

```diff
@@ -238,6 +238,50 @@ function math.sign(x, tolerance)
 end
 
 --------------------------------------------------------------------------------
+-- Video enums and packing function
+
+-- E_BLEND_FACTOR
+core.ebf = {
```

Comment:
@Foghrye4 In no other part of the API are "enums" like these used. In all parts that use "enums", an `EnumString` or similar is used.

It's more consistent both with the rest of Minetest and with the way this is done normally in Lua (see e.g. `file:seek` and `collectgarbage`).

Some thing like:

```
local blend_modes = {
  foo = 1,
  bar = 2,
  -- ...
}

function pack_whatever(foo, bar)
  foo = assert(blend_modes[foo], "invalid foo!")
  bar = assert(blend_modes[bar], "invalid bar!")
  return 42 + foo*bar ^ 1337
end
```